src/components/Hero/Hero.mdx

```diff
@@ -198,6 +198,48 @@ subHeaderText={otherSubHeaderText}
 
 <Canvas of={HeroStories.CampaignBackgroundColors} />
 
+## Fallback Campaign Image
+
+<Source
+  code={`
+const FallBackExample = () => {
+  // Or get image src from external service.
+  const [imageSrc, setImageSrc] = useState("foobar.jpg");
+  const fallbackImageSrc = "https://loremflickr.com/cache/resized/65535_51121136167_40f20af042_z_300_300_nofilter.jpg";
+  // For some reason IIIF image as fallback does not work...
+  // Perhaps the better approach is to store the image locally in the app.
+  // This cannot be easily tested in Storybook.
+  // const fallbackImageSrc = "https://iiif.nypl.org/iiif/2/105224/full/!900,900/0/default.jpg";
```

Comment:
Not sure why this fails but if you can try locally to verify, that'd be great. I don't think it has to do with the DS or how images are loading but maybe instead iiif? loremflicker and other images that are used as fallback images load.

Reply:
Seeing the same issue locally, and images not from IIIF do load